content/administration/install.rst

```diff
@@ -107,18 +107,7 @@ Linux
 Debian/Ubuntu
 '''''''''''''
 
-Odoo 13.0 'deb' package currently supports `Debian Buster`_, `Ubuntu 18.04`_ or above.
-
-Prepare
-^^^^^^^
-
-Odoo needs a `PostgreSQL`_ server to run properly. The default configuration for
-the Odoo 'deb' package is to use the PostgreSQL server on the same host as your
-Odoo instance. Execute the following command in order to install the PostgreSQL server:
-
-.. code-block:: console
-
-  $ sudo apt install postgresql -y
+Odoo 14.0 'deb' package currently supports `Debian Buster`_, `Ubuntu 20.04`_ or above.
```

Comment:
`.../content/administration/install.rst:110: WARNING: Unknown target name: "ubuntu 20.04".`

FYI `Ubuntu 20.04`_ is a link and should also have been specified/updated.
I'll fix it in another PR ;).

Reply:
Oops, you are right.

Reply:
#963 should fix that